api/config/module/chaos/experimentation/xds/v1/xds.proto

```diff
@@ -26,4 +26,10 @@ message Config {
 
   // The interval at which to send heartbeat responses for TTL'd resources.
   google.protobuf.Duration heartbeat_interval = 6;
+
+  // list of clusters for which ECDS is enabled. * means it will be enabled for all
+  repeated string ecds_enabled_clusters = 7;
```

Comment:
an alternative API that avoids the need for special strings would be 

```
message ECDSAllowList {
  repeated enabled_clusters = 1;
}

// Specifies which clusters ECDS is enabled for, causing faults to be communicated over ECDS instead of RTDS where supported. If unset, all ECDS will be used for all clusters.
ECDSAllowList ecds_allow_list = 7;

```